themes/grav/templates/forms/fields/editor/editor.html.twig

```diff
@@ -5,8 +5,10 @@
 
 {% block label %}
     {% if field.label %}
-        {% set hint = field.help ? 'data-hint="' ~ field.help|tu|raw ~ '"': '' %}
```

Comment:
This is crazy, and I have absolutely no idea what's causing it - if anyone more familiar with Twig knows, I'd be really interested in hearing!

I noticed the help text wasn't appearing correctly, and the HTML generated for the `data-hint` looked like `data-hint=""Default" headers to add to new pages&quot;`

It looks like Twig processes strings weirdly when surrounded in quotes.

For example, the code before this diff, and this following sample:

```twig
{{ "\"" ~ field.help ~ "\"" }}
```
if `field.help` is `"some test"`, will produce the output `""some" test&quot;`

Whereas

```twig
"{{ field.help }}"
```
correctly generates `"some test"`

No idea why... So this just side-steps the issue